REFACTOR-#4513: Fix spelling mistakes in docs and docstrings (#4514)
modin-project/modin: docs/getting_started/troubleshooting.rst

be set fine-grained as it possible (specify ``dtype`` parameter only for columns with
heterogeneous data).

> Setting of ``dtype`` parameter works well for most of the cases, but, unfortunately, it is

Reviewer: This seems a bit odd grammar-wise. Would do something like "[...] for most of the cases; unfortunately, it is [...]"

Reviewer 2: ```suggestion
Specifying the ``dtype`` parameter will work well in most cases. Unfortunately, it is
```

> ineffective if data file contain column which should be interpreted as index

Reviewer 2: ```suggestion
ineffective if the data file contains a column which should be interpreted as the index
```

    with Pool(5) as p:
        print(p.map(f, [1]))

> Even if this example may work on your machine, we do not recommend it.

Reviewer 3: ```suggestion
Although this example may work on your machine, we do not recommend it, because
```

> The Python multiprocessing will cause conflicts with excessive resource use

Reviewer 3: ```suggestion
the Python multiprocessing library will duplicate Ray clusters, causing both excessive resource usage and conflict over the available resources.
```

**Solution**

> To avoid the problem Dask Client creation needs to be moved into the ``__main__`` scope of the module.

Reviewer 3: ```suggestion
To avoid the problem the Dask Client creation code needs to be moved into the ``__main__`` scope of the module.
```
